graphein/molecule/atoms.py

```diff
@@ -0,0 +1,34 @@
+"""
+Author: Eric J. Ma, Arian Jamasb
+Purpose: This is a set of utility variables and functions that can be used
+across the Graphein project.
+
+These include various collections of standard & non-standard/modified amino acids and their names, identifiers and properties.
+
+We also include mappings of covalent radii and bond lengths for the amino acids used in assembling atomic protein graphs.
+"""
+# Graphein
+# Author: Eric J. Ma, Arian Jamasb <arian@jamasb.io>
+# License: MIT
+# Project Website: https://github.com/a-r-j/graphein
+# Code Repository: https://github.com/a-r-j/graphein
+
+
+from typing import Dict, List
+
+import numpy as np
+
+BASE_ATOMS: List[str] = [
```

Comment:
We may want to have some different sets of allowable atoms. I can check some previous projects for the sets we allowed there

Reply:
I also checked this, OGB basically used atomic numbers from 1-120 and leave others as others. Feel free to change!